src/lib/components/pages/anime_info/error.svelte

```diff
@@ -35,11 +35,13 @@
 
     <section
         transition:blur
-        class:md:flex-row-reverse={on_left}
-        class:md:flex-row={on_right}
-        class={cn(mapping.class, "relative flex h-full flex-col justify-end gap-20 md:items-end md:gap-0")}
+        class={cn(mapping.class, "relative grid h-full grid-flow-col justify-end gap-20 md:col-span-2 md:gap-0")}
     >
-        <error-context class="flex flex-col items-center leading-none md:mb-[13vw] md:w-[70dvw] md:items-start md:gap-[1vw] md:pl-[5vw]">
+        <error-context
+            class:order-1={on_right}
+            class:order-2={on_left}
```

Comment:
This is what you had to do

Reply:
https://github.com/tokitou-san/CoreProject-Frontend/pull/760/files#r1327920187